### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by suyash-20**
1. Integrated class-based print methods.
2. Choice variable is assigned with Emun type for better readability
3. Took care of the initial if-else condition.

### CODE REVIEW

File: application1.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+    if choice == MainChoice.Books:
+        print("""
+        1. Browse through the book catalogue.
+        2. Add books to the catalogue.
+		""")
+		
+        book_input = int(input("Enter your choice as '1' or '2': "))
+        
+        if book_input == 1:
```
Comment (ulhas) [inline application1.py]: Convert this to enum as well

File: application1.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            print("Here's our Book catalogue:")
+            for row in book_info:
+                book = Book(row[0],row[1],row[2],row[3])
+                book.print_books()
```
Comment (ulhas) [inline application1.py]: print_books is redundant. when you are giving print statement, it's already for a book object. no need to add `_books` again. it's understood.

**Review by ulhas — CHANGES_REQUESTED** [review]

File: application1.py (hunk 2 above)
Comment (suyash-20) [reply application1.py]: Okay, will revert it back to print

File: application1.py (hunk 1 above)
Comment (suyash-20) [reply application1.py]: Will do that.

**Review by suyash-20 — COMMENTED** [review]
Apart from print method name and enum declarations, are there any other changes required? @ulhas

**Pull request merged**